dpdata/plugins/openmx.py

```diff
@@ -0,0 +1,36 @@
+import dpdata.md.pbc
+import dpdata.openmx.omx
+from dpdata.format import Format
+
+@Format.register("openmx")
+class OPENMXFormat(Format):
```

Comment:
@njzjz Thank you for the reply.

I have a question. What is the “documentation" in your message?

For example, on page 36 of the dpdata documentation, there are description about `qe/cp/traj`, right? Do you meant that a similar description should be added for `openmx`?

Or is it sufficient to simply attach a link to the DFT package OpenMX, like [this](https://www.openmx-square.org/)?

I look forward to your advice.

Reply:
Something like this, e.g. what openmx is and what this format is.

https://github.com/deepmodeling/dpdata/blob/12df3484a98e7233df4d48b34d638ea570575a47/dpdata/plugins/ase.py#L20-L28

Btw, it may be better to rename it to `openmx/out` (or something like this) to show this is the output but not the input. The input file can also be a format.

Reply:
@njzjz Thanks.

I maked out what you meant. So, I added some documentation on this new format, and change keyword from `openmx` to `openmx/out`.